### DISCUSSION TITLE
Internet Explorer Polyfilling

### DISCUSSION THREAD

**Discussion by akeohanePing** [adobe/react-spectrum#4381] [General]
Hey there,

We have a client running an office 365 implementation of our product -- at one point there is a browser that opens embedded and it is Internet Explorer 11.

React Spectrum does not support IE. One of our platforms uses hooks from React Aria and React Stately. This causes the bundle to not load in IE.

My team is troubleshooting potential solutions to this issue.

Is it even worth looking into polyfilling React Spectrum libraries?
Does anyone have experience trying?
What would be a good place to start?

**Comment by snowystinger** [comment]
I don't think it's worth looking into. Instead I'd follow microsofts official site for end of life
https://learn.microsoft.com/en-us/lifecycle/faq/internet-explorer-microsoft-edge which has this link on migrating Office 365
https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-365-apps-say-farewell-to-internet-explorer-11-and/ba-p/1591666 this timeline has been finished for a year and a half
